### DISCUSSION TITLE
请教如何把配置从一个环境迁移到另一个环境

### DISCUSSION THREAD

**Discussion by gotnix** [apolloconfig/apollo#5168] [Q&A]
Apollo 使用的 1.6.2，使用过程中拆分了不同的 ENV，有一批应用想把原来 ENV 的配置迁移到新的 ENV 中，除了在 Portal 中手动复制粘贴 TXT ，目前想到调接口或者直接写 SQL 导数据，还有没有其他更好的方法？

如果哪位大佬实践过，有没有什么注意事项？

先行谢过！

**Comment by nobodyiam** [comment]
新版本有配置导入和导出的功能，可以参考下

![image](https://github.com/apolloconfig/apollo/assets/837658/36555d0b-e73d-4695-8721-7d949e7290d1)

**Reply by gotnix** [reply]
感谢回复！

暂时不方便升级版本，使用的客户端五花八门不知道升级之后是否兼容。无意间发现每个 Namespace 右上角都有一个『同步配置』的按钮，感觉是我想要的功能，我先测试一下。之前简直是眼瞎了。
![Appllo_Namespace_同步各环境间配置](https://github.com/apolloconfig/apollo/assets/576470/bb43570f-6508-4738-b935-d6f3e614e89f)
